doc/user/content/sql/drop-index.md

```diff
@@ -69,6 +76,33 @@ SHOW INDEXES FROM q01;
 DROP INDEX materialize.public.q01_geo_idx;
 ```
 
+### Drop indexes and dependent objects
+
+```sql
+SHOW VIEWS;
+```
+```nofmt
++-----------------------------------+
+| VIEWS                             |
+|-----------------------------------|
+| ...                               |
+| q01                               |
++-----------------------------------+
+```
+```sql
+SHOW INDEXES FROM q01;
+```
+```nofmt
++------------------------+--------------------------------+-----
+| View                   | Key_name                       | ...
+|------------------------+--------------------------------+----
+| materialize.public.q01 | materialize.public.q01_geo_idx | ...
++------------------------+--------------------------------+-----
+```
+```sql
+DROP INDEX materialize.public.q01_geo_idx CASCADE;
```

Comment:
We don't typically include the full qualification in these examples, which is a potentially instructive point that index names are referrable by the same rules as all other catalog objects.